app/src/main/java/org/simple/clinic/settings/SettingsEffectHandler.kt

```diff
@@ -12,12 +12,14 @@ object SettingsEffectHandler {
   fun create(
       userSession: UserSession,
       settingsRepository: SettingsRepository,
-      schedulersProvider: SchedulersProvider
+      schedulersProvider: SchedulersProvider,
+      uiActions: UiActions
```

Comment:
Make `schedulersProvider` the last parameter?

Reply:
Done.